### PR TITLE
completed javascript-objects

### PR DESCRIPTION
**Pull request by kyle-foral**
image

### CODE REVIEW

File: javascript-objects/main.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+};
+const fullName = student.firstName + ' ' + student.lastName;
+console.log('The value of fullName ', fullName);
+
+student.livesInIrvine = false;
+student.previousOccupation = 'Solar Installer';
+
+console.log('The value of livesInIrvine ', student.livesInIrvine);
+console.log('The value of previousOccupation ', student.previousOccupation);
+
```
Comment (shawnkost) [inline javascript-objects/main.js:14]: You should also be console logging the whole `student` object here

**Review by shawnkost — CHANGES_REQUESTED** [review]
One small change requested Kyle!

File: javascript-objects/main.js (hunk 1 above)
Comment (kyle-foral) [reply javascript-objects/main.js:15]: image

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**